### PR TITLE
Add 'altText' to all sample cards

### LINKED ISSUE #6421: [Website] Image Gallery: Alt is not defined for images under Image Gallery adaptive card.
### Target Platforms

NodeJS

### SDK Version

9/5/2021

### Application Name

https://adaptivecards.io/samples/ImageGallery.html

### Problem Description

[ADO 35800985](https://microsoft.visualstudio.com/OS/_workitems/edit/35800985)

**Repro steps:**
1.Open above URL on Edge Dev and Enable Narrator.
2.Navigate to Image gallery and activate it.
3.Navigate to imaged by using up and down arrow key in scan mode and verify the issue.

**Actual Result:**
Alt is not defined for images under Image Gallery adaptive card. Screen reader only announces 'Image'.

**Expected Result:**
Alt should be properly defined for the images under image gallery card, so that screen reader should understand the control. 

### Screenshots

_No response_

### Card JSON

```JSON
ImageGallery.json
```

### Sample Code Language

_No response_

### Sample Code

_No response_

### LINKED ISSUE #6424: [Website] Schema Explorer: Ensures <img> elements have alternate text or a role of none or presentation.
### Target Platforms

NodeJS

### SDK Version

https://adaptivecards.io/  9/9/2021

### Application Name

Website

### Problem Description

[ADO 35903425](https://microsoft.visualstudio.com/OS/_workitems/edit/35903425)

**Repro Steps:**
1. Open above URL with valid credential.
2. Turn on the Narrator using ctrl+win+enter key.
3. Navigate to 'Schema Explorer' link and using arrow keys press enter key to activate it.

**Actual Result:**
Alt is not defined for 'Decorative image' under 'Publish Adaptive Card Schema.'

**Expected Result:**
Alt should be defined for the decorative image under 'Publish Adaptive Card Schema.'

### Screenshots

_No response_

### Card JSON

```JSON
No card
```

### Sample Code Language

_No response_

### Sample Code

_No response_

### LINKED ISSUE #6426: [Website] Activity Update: Ensures elements with an ARIA role that require parent roles are contained by them (aria-required-parent -
### Target Platforms

NodeJS

### SDK Version

https://adaptivecards.io/  9/9/2021

### Application Name

Website

### Problem Description

[ADO 35919222](https://microsoft.visualstudio.com/OS/_workitems/edit/35919222)

**Repro Steps:**
1. Open above URL with valid credential.
2. Navigate to 'Samples' link and press tab key to activate it.
3. Select the 'Activity Update' option and navigate to 'Date Picker' under 'Publish Adaptive Card Schema'.
4. Run Accessibility insight for Web and observe the issue.

**Actual Result:**
Ensures elements with an ARIA role that require parent roles are contained by them.

**Expected Result:**
Ensures elements with an ARIA role that require parent roles are contained by them.

### Screenshots

_No response_

### Card JSON

```JSON
ActivityUpdate.json
```

### Sample Code Language

_No response_

### Sample Code

_No response_

### PR DESCRIPTION
**Pull request by carlos-zamora**
# Related Issues
- Closes #6424 
- Closes #6421 

# Description
Ensures `"altText"` is defined for all sample cards located in the `samples` folder. This includes template and non-template cards.

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/AdaptiveCards/pull/7102)

### CODE REVIEW

File: samples/v1.1/Tests/ImageSet.Stretch.json [hunk 1]
```diff
@@ -1,907 +1,1046 @@
...
-            "text": "ImageSet: ImageSize: small"
-        },
-        {
-            "type": "ImageSet",
-            "imageSize": "small",
-            "images": [
-                {
-                    "type": "Image",
-                    "backgroundColor": "#FF0000FF",
-                    "url": "https://adaptivecards.io/content/cats/1.png"
```
Comment (carlos-zamora) [inline samples/v1.1/Tests/ImageSet.Stretch.json:17]: ⭐**Pro Tip**⭐
View this file using the "hide whitespace changes" option in GitHub's reviewing tool. For some reason, the diff appears as the entire file even though the tabs were maintained (might be the line endings? idk).

File: samples/v1.5/Test/RtlTestCard.json [hunk 2]
```diff
@@ -179,12 +181,14 @@
         {
           "type": "Image",
           "url": "https://adaptivecards.io/content/cats/1.png",
+          "altText": "cat",
```
Comment (PankajBhojwani) [inline samples/v1.5/Test/RtlTestCard.json:184]: Tiny nit but any reason the first letter isn't capitalized here but it is for the other alt texts?

Comment (carlos-zamora) [reply samples/v1.5/Test/RtlTestCard.json:184]: No reason. Good catch. I'll fix it.

**Review by PankajBhojwani — APPROVED** [review]

**Review by paulcam206 — APPROVED** [review]
so many cats!

:shipit:

**Review by beervoley — APPROVED** [review]

**Pull request merged**